src/topology/continuous_function/compact.lean

```diff
@@ -45,6 +46,36 @@ equivalent to `C(α, β)`.
 def equiv_bounded_of_compact : C(α, β) ≃ (α →ᵇ β) :=
 ⟨mk_of_compact, to_continuous_map, λ f, by { ext, refl, }, λ f, by { ext, refl, }⟩
 
+lemma uniform_inducing_equiv_bounded_of_compact :
+  uniform_inducing (equiv_bounded_of_compact α β) :=
+uniform_inducing.mk'
```

Comment:
I wonder is there a proof of this using the language of filters a bit more and appealing to `has_basis_compact_convergence_uniformity` rather than `mem_compact_convergence_entourage_iff`. Something like:
```lean
⟨begin
  symmetry,
  apply has_basis_compact_convergence_uniformity.eq_of_same_basis,
  -- your proof here!
end⟩
```

Reply:
I'm looking into this.